src/components/templates/Page.tsx

```diff
@@ -21,9 +22,15 @@ export default function Page({
   headerCenter
 }: PageProps): ReactElement {
   const isHome = uri === '/'
+  const isSearch = uri === '/search'
 
   const childElements = (
     <>
+      {isSearch && (
+        <>
+          <SearchBar visibleInput placeholder="Search for service offerings" />
```

Comment:
nitpick(blocking): Can we get this placeholder value from a central place? We will forget to update this if we write a custom text here.